listener.go

```diff
@@ -79,6 +92,7 @@ func (h *hub) makeDumpResponses() (msgs []*baps3.Message) {
 		msgs = append(msgs, baps3.NewMessage(baps3.RsTime).AddArg(
 			strconv.FormatInt(h.downstreamState.Time.Nanoseconds()/1000, 10)))
 	}
+	msgs = append(msgs, h.makeRsAutoadvance())
```

Comment:
The lowercase `a` in `Autoadvance` makes me twitch a little, but I guess otherwise it'd sound like some function that makes an `Rs` auto-advance.  Hm.